core/unix/signal_linux.c

```diff
@@ -363,15 +363,22 @@ handle_pre_extended_syscall_sigmasks(dcontext_t *dcontext, kernel_sigset_t *sigm
      */
     if (sizemask != sizeof(kernel_sigset_t))
         return false;
+    /* Fixes #3344. */
```

Comment:
Is this trying to trigger the issue tracker?  It needs to be in the commit message.  When I said "here" in the prior review comment I meant in this PR/commit as opposed to manually closing the issue separately after merging.

Reply:
Ok, I was surprised you wanted to add this here. Added to commit message instead.